### DISCUSSION TITLE
无法创建Java虚拟机

### DISCUSSION THREAD

**Discussion by 1jntm1** [Meloong-Git/PCL#1431] [提问] [answered]
为啥我在打开游戏时会提示“无法创建java虚拟机”有解决方法吗

**Comment by hejiehao** [comment]
> 为啥我在打开游戏时会提示“无法创建java虚拟机”有解决方法吗

发日志或截图

**Reply by 1jntm1** [reply]
日志：Invalid maximum heap size: -Xmx7372m
The specified size exceeds the maximum representable size.

**Reply by 1jntm1** [reply]
![屏幕截图 2023-03-11 171510](https://user-images.githubusercontent.com/126381605/224476030-3830bb30-7239-4daf-9998-1f4925bafbd7.png)

**Reply by hejiehao** [reply]
> ![屏幕截图 2023-03-11 171510](https://user-images.githubusercontent.com/126381605/224476030-3830bb30-7239-4daf-9998-1f4925bafbd7.png)

重新安装java

**Reply by hejiehao** [reply]
我一般推荐bellsoft的java

**Comment by WForst-Breeze** [comment]
请尝试配置 Java 环境变量后再次进行相关操作，并查看问题是否存在

**Comment by IF-desu** [comment]
尝试切换其他版本的java

**Comment by IF-desu** [comment]
检查是否使用了32位java，是的话切换成64位java

**Comment by LTCatt** [comment, marked as answer]
```
Invalid maximum heap size: -Xmx7372m
The specified size exceeds the maximum representable size.
```

如果是这个报错的话可以交一个 issue
